### ISSUE TITLE
How to get all pages for /activities endpoint

### ISSUE THREAD

**Issue by urgas9** [MicrosoftDocs/CloudAppSecurityDocs#418]
I am trying to list all activities using `/api/v1/activities` endpoint. No luck so far. I am trying to use `skip` to utilize pagination:
```
{
    "sortDirection": "desc",
    "sortField": "date",
    "limit": 1,
    "skip": 10
}
````

[According to the docs](https://docs.microsoft.com/en-us/cloud-app-security/api-activities-investigate-script#response-parameters), I should use `nextQueryFilters` but even though I get `hasNext` set to `true` in the API response I get no `nextQueryFilters` (I only get `data`, `hasNext` and `total`).

Would someone be able to help me out and make the pagination work (list all activities) and also make the documentation clearer? 

Thanks

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7176c15a-025f-f67a-502c-5353eced167f
* Version Independent ID: ec00e101-7d22-d7b6-f6a7-1fc2208eb69a
* Content: [Cloud App Security REST API](https://docs.microsoft.com/en-us/cloud-app-security/api-introduction)
* Content Source: [CloudAppSecurityDocs/api-introduction.md](https://github.com/Microsoft/CloudAppSecurityDocs/blob/master/CloudAppSecurityDocs/api-introduction.md)
* Service: **cloud-app-security**
* GitHub Login: @dcurwin
* Microsoft Alias: **dacurwin**

**Comment by dcurwin** [comment]
Thanks for your question. We'll investigate and get back to you.

**Comment by dcurwin** [comment]
@urgas9 - I contacted the product group and this is their response:

> We recommend you use https://docs.microsoft.com/en-us/cloud-app-security/api-activities-investigate-script and alter the filters according to your needs. Alternatively, use https://docs.microsoft.com/en-us/cloud-app-security/api-activities-list with the skip parameter set to 0, and the limit parameter set to the desired amount of records. 

Is that helpful?

**Comment by urgas9** [comment]
I already checked that page - the problem is that there is no `nextQueryFilters` field in the response body in my case.

However, I did manage to make it work using `created` field and order descending and then iterating using lower `created` values. Also, I stumbled pretty hard requesting `POST /activities` where the filter did not work - it did start working using `POST /activities/` (mind the `/` at the end).
